### ISSUE TITLE
Error When using enact create

### ISSUE THREAD

**Issue by ascendsauce** [enactjs/cli#225]
I get the following error when trying to execute `enact create test`. The error does not occur if I revert to @enact/cli@2.6.0.

Creating a new Enact app in test...

npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/enyojs/sandstone.git
npm ERR! 
npm ERR! Warning: Permanently added the RSA host key for IP address *** to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128

**Comment by mattylase** [comment]
I'm also running into this! Any input or fix would be great.

**Comment by webOS101** [comment]
Thanks for the report!  As a quick workaround, you can do the following:
```
enact template install @enact/template-moonstone
enact create -t moonstone test
```

**Comment by JayCanuck** [comment]
@ascendsauce this should be fixed as of the latest Enact CLI 3.0.4 release. Feel free to create a followup issue if it (or other issues) occur again.
